scss/_functions.scss

```diff
@@ -50,3 +50,41 @@
 		@return false;
 	}
 }
+
```

Comment:
Could we add a note that this is temporary? Until the next breaking change, and somehow discourage its direct use, maybe?

Reply:
Direct use is discouraged because it is private. Good shout on making a note to remove it on the next breaking. :) https://github.com/Financial-Times/o-buttons/issues/156